apps/chef_db/itest/itest_util.erl

```diff
@@ -28,23 +28,16 @@
 -include("../../include/chef_types.hrl").
 
 create_record(Record) ->
```

Comment:
minor, but if we're just making this a simple passthrough, maybe we could eliminate this module call directly? (For the others below as well)